test/unit/cog_api/fake/roles_test.exs

```diff
@@ -98,4 +98,17 @@ defmodule CogApi.Fake.RolesTest do
       assert first_role.id == role.id
     end
   end
+
+  describe "role_revoke" do
+    it "returns the roles that are associated with that group" do
+      role = Client.role_create(fake_endpoint, %{name: "role123"}) |> get_value
+      group = Client.group_create(fake_endpoint, %{name: "group123"}) |> get_value
+      group = Client.role_grant(fake_endpoint, role, group) |> get_value
```

Comment:
Not sure I"m following. I have those phases split up with the line breask currently. Lines 104-107 are all setup (the `assert` there is just a check to make sure our setup is correct).

Line 109 is the actual exercise.

Line 111 is the verification.